server/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -1274,26 +1257,6 @@ private void assertDuplicatePrimaryKey(String stmt) {
         }
     }
 
-    @Test
-    public void testAlterTableAddColumnWithCheckConstraint() throws Exception {
```

Comment:
We cover such scenarios in `DDLIntegrationTest`, see 
`test_alter_table_add_column_succedds_because_check_constaint_refers_to_self_columns`
 and 
`test_add_column_all_supported_configs_applied_to_altered_table`.

And now we cannot check mapping on alter table since we don't copy mapping anymore, we just send delta